az-digital/az_quickstart
# Closes #2239: Remove unnecesssary timezone processing from Trellis Events importer.

## Description
The event datetime values in the Trellis events API are UTC values so the timezone processing steps we are doing are unnecessary (and problematic).


## Related issues




Closes #2239

## How to test


- Import a Trellis Event
- Verify that the event datetime matches what it is in Trellis

## Types of changes


### Arizona Quickstart (install profile, custom modules, custom theme)
- **Patch release changes**
- **Minor release changes**
- **Other or unknown**

### Drupal core
- **Patch release changes**
- **Minor release changes**
- **Other or unknown**

### Drupal contrib projects
- **Patch release changes**
- **Minor release changes**
- **Other or unknown**

## Checklist
